content/enterprise_influxdb/v1.9/administration/manage/clusters/_index.md

```diff
@@ -0,0 +1,21 @@
+---
+title: Manage InfluxDB Enterprise clusters
+description: >
+  Use the `influxd-ctl` and `influx` command line tools to manage InfluxDB Enterprise clusters and data.
+aliases:
+    - /enterprise/v1.8/features/cluster-commands/
+    - /enterprise_influxdb/v1.9/features/cluster-commands/
+    - /enterprise_influxdb/v1.9/administration/cluster-commands/
+menu:
+  enterprise_influxdb_1_9:
+    name: Manage clusters
+    weight: 30
+    parent: Manage
+---
+
+Use the following tools to manage and interact with your InfluxDB Enterprise clusters:
+
+- [`influxd-ctl`](/enterprise_influxdb/v1.9/tools/influxd-ctl/) cluster management utility
+- [`influx`](/enterprise_influxdb/v1.9/tools/influx-cli/use-influx/) command line interface (CLI)
+
+{{< children >}}
```

Comment:
let's keep menu and children order the same --prob makes sense to start w add node